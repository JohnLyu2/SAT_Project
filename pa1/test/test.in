1 * VAr ; VAr: 1, VAr: 0
1 * VAr ; VAr: 1
1 * VAr;VAr : 0
1 * VAr + 0
1 * VAr + 0 ; VAr 1
1 *(VAr + 0 );
A + B * -0 * (1 + -CD); A : 1, B : 0, CD : 1
A + B * -0 * (1 + -CD); A : 0, B : 0, CD : 1
   Ur * c; Ur:1, c:1
 Ur * c * (1+0)) ; Ur:1, c:1
 Ur * c * (1+0) ; Ur:1, c:1
 (1 * (a1 +VAr)) ; VAr: 1, a1: 0, VAr: 0
 (1 * (a1 +VAr)) !; VAr: 1, a1: 0, VAr: 1
1*0+-1;
 (1 * (0 +1)) ; 
