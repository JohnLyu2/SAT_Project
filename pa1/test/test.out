Error: contradicting assignment
1
0
Error: invalid input
Error: invalid input
Error: incomplete assignment
1
0
1
Error: invalid input
1
Error: contradicting assignment
Error: invalid input
0
1
